src/views/connecting/Connecting.js

```diff
@@ -101,14 +101,28 @@ export const Connecting = (props) => {
 
     const statusChanged =
       pollingState.previousResponse?.connection_status !==
-      pollingState.currentResponse?.connection_status
+        pollingState.currentResponse?.connection_status ||
+      pollingState.previousResponse?.raw_status !== pollingState.currentResponse?.raw_status
 
     // if status changes during connecting or timeout send out a post message
     if (pollingState.previousResponse != null && statusChanged) {
-      postMessageFunctions.onPostMessage('connect/memberStatusUpdate', {
+      const event = {
```

Comment:
I would rename this `metadata` or `eventMetadata` as it's more what this object is.